src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -38,6 +39,8 @@ public partial class LibraryListView : UserControl
 		private bool _primaryCollectionReloadPending;
 		private bool _disposed;
 		private BookCollection _downloadedBookCollection;
+		private PictureBox _bookTriangle;
```

Comment:
I'm not keen on this name. Took me quite a while to figure out that this was an icon to provide an alternative way to get at (part of) the right-click menu, and that it moves next to the active thumbnail. Would _contextMenuButton be better? Even _thumbnailContextMenuButton? It's not used so much that a long informative name is inappropriate. A comment summarizing what it is for might be helpful, too.